### DISCUSSION TITLE
Goto matching bracket (`mm`) in comment jump to the wrong place

### DISCUSSION THREAD

**Discussion by Luv-Ray** [helix-editor/helix#10986] [General]
### Summary

In this code:
```rust
enum E {
    /// [test]
    A(i32),
}
```
put cursor on `[`, and call `mm`, the cursor will jump to the `}` at the end line, not `]`.

### Helix log

no log

### Platform

macOS

### Terminal Emulator

warp-terminal v0.2024.06.11.08.02.stable_03

### Installation Method

source

### Helix Version

helix 24.3 (dbacaadd)

**Comment by pascalkuthe** [comment]
This is intentional behavior. `mm` is tree sitter based. The bracket in this comment is not part of the grammar and just plaintext
